db/schema.rb

```diff
@@ -166,6 +206,15 @@
     t.index ["ukprn"], name: "index_local_authorities_on_ukprn"
   end
 
+  create_table "outcomes", force: :cascade do |t|
+    t.string "state"
+    t.date "completion_date"
```

Comment:
add `null: false`

Reply:
think about naming for another time: should this be consistent with `_on` or `_date`